_observing-your-data/alerting/monitors.md

```diff
@@ -138,7 +138,10 @@ POST _nodes/reload_secure_settings
 1. Specify a name for the monitor.
 1. Choose either **Per query monitor**, **Per bucket monitor**, **Per cluster metrics monitor**, or **Per document monitor**.
 
-Per query monitors run your specified query and then check whether the query's results trigger any alerts. Per bucket monitors let you select which fields to create buckets and categorize your results into those buckets. The Alerting plugin runs each bucket's unique results against a script you define later, so you have finer control over which results should trigger alerts. Each of those buckets can trigger an alert, but query-level monitors can only trigger one alert at a time.
+Per query monitors run your specified query and then check whether the query's results trigger any alerts. Per bucket monitors let you select which fields to create buckets and categorize your results into those buckets. The Alerting plugin runs each bucket's unique results against a script you define later, so you have finer control over which results should trigger alerts. Each of those buckets can trigger an alert, but query-level monitors can only trigger one alert at a time. The maximum number of monitors you can create is 1,000 and is configurable if you call the cluster settings API `plugins.alerting.monitor.max_monitors` to change the settings for the maximum monitor count.
```

Comment:
```suggestion
Per query monitors run your specified query and then check whether the query's results trigger any alerts. Per bucket monitors let you select which fields to create buckets and categorize your results into those buckets. The Alerting plugin runs each bucket's unique results against a script you define later, so you have finer control over which results should trigger alerts. Each bucket can trigger an alert, but the query-level monitors can only trigger one alert at a time. The maximum number of monitors you can create is 1,000 and is configurable if you call the cluster settings API `plugins.alerting.monitor.max_monitors` to change the settings for the maximum monitor count.
```
```suggestion
Per query monitors run your specified query and then check whether the query's results trigger any alerts. Per bucket monitors let you select which fields to create buckets and then categorize your results into those buckets. 

The Alerting plugin runs each bucket's unique results against a script you define later, so you have finer control over which results should trigger alerts. Each bucket can trigger an alert, but query-level monitors can only trigger one alert at a time. The maximum number of monitors you can create is 1,000 and is configurable if you call the cluster settings API `plugins.alerting.monitor.max_monitors` to change the settings for the maximum monitor count.
```

Reply:
I would reformat this section for readability, using the wording changes proposed by Melissa:

```suggestion

OpenSearch supports the following types of monitors:

- **Per query monitors** run your specified query and then check whether the query's results trigger any alerts. Query-level monitors can only trigger one alert at a time.
- **Per bucket monitors** let you select which fields to create buckets and categorize your results into those buckets.  The Alerting plugin runs each bucket's unique results against a script you define later, so you have finer control over which results should trigger alerts. Furthermore, each bucket can trigger an alert.

The maximum number of monitors you can create is **1,000**. If you want to change the default maximum number of alerts for your cluster to below 1000, you can is configure the maximum by calling the cluster settings API `plugins.alerting.monitor.max_monitors`.
```